Re-organize Components (#1643)
near/docs: docs/bos/dev/intro.md

---
id: intro
title: Dev Environments
---

Building, deploying, and collaborating within a new framework can be challenging. The NEAR community has created helpful workflows using new tools to simplify your development workflow.

The environments in which you can develop `Components` are divided into two categories:

- [Web Tools](#web-tools): Online tools that allow you to quickly start building and sharing components.
- [Local Tools](#local-tools): Tools that allow you to develop and test components locally.

:::tip Handling collaborations

> If you plan to collaborate with multiple people, we recommend you to check [our repo template](https://github.com/near-examples/bos-components) which automatically deploys components each time a PR is merged. 

Reviewer: ```suggestion
If you plan to collaborate with multiple people, we recommend you to check [this repository template](https://github.com/near-examples/bos-components) which automatically deploys components each time a PR is merged. 
```

:::

---

## Web Tools

> The quickest way to start building NEAR Components is by using one of the online tools. They are great to take your first steps, while they help to find other developers and components in the NEAR ecosystem.

Reviewer: ```suggestion
The quickest way to start building NEAR Components is by using one of the online tools. They are great for taking your first steps while they let you find other developers and components in the NEAR ecosystem.
```
